modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,69 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by Organization Administrators, developers, and end-users; each of these roles has different privileges when connecting applications,
+and there are some restrictions to ensure sensitive data remains secure.
+
+== Organization Administrator
+
+An Organization Administrator is the only role that can view and manage Connected Apps in Access Management.
+
+To manage your connected apps, navigate to Access Management and select the *Applications* tab.
+
+// screenshot of Connected Apps page
+
+The *Connected Apps* page shows a list of applications that you have connected to Anypoint Platform.
+To add an application, select *Add Application*.
+
+=== Create Application
+
+When you create an application as an Organization Administrator, consider the following:
+
+* An organization can own up to 200 connected apps.
+* You can disable or whitelist connected apps.
+* An application is assigned a clientID upon creation; the clientID cannot be modified. However, client secrets can be modified in the *Application Settings* page.
+* When you create an application with the client grant type, the application will continue to work if the application owner's credentials are disabled for any reason. A team or company can assume control of the application's access.
+* An application's actions are logged with an application ID and user ID (for user grant type applications only) in the Audit Log. This empowers administrators to monitor and audit data use.
+
+// screenshot of Create Application interface
+
+When creating an application, you must enter the following information:
+
+* *Application Name*: A descriptive name for your application.
+* *Redirect URIs*: Uniform Resource Identifier that specifies the location of the application. You can specify one per line.
+* *User Grant Types* or *Client Grant Types*: A grant type that aligns with OAuth 2.0 specifications.
```

Comment:
This looks like UI, so there must be a user grant type. But please do verify what that means because, again, in the world of OAuth 2.0 I don't know of a "user" grant type. Not that I'm an expert.

Reply:
Just verify that if it is indeed a "user" grant type, does that mean the user name and password are passed, or...?